Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -691,6 +694,134 @@ public class CameraManager {
     }
 }
 
+// MARK: Camera Animation
+extension CameraManager: CameraAnimatorDelegate {
+    // MARK: Animator Functions
+
+    /**
+     Convenience to create a `CameraAnimation` and will add it to a List of `CameraAnimators` to track the lifecycle of that animation.
+
+     - Parameter duration: The duration of the animation, in seconds.
+     - Parameter timingParameters: The object providing the timing information. This object must adopt the `UITimingCurveProvider` protocol.
+     - Parameter animationOwner: The `id` of what owns this animation.
+     - Returns `CameraAnimator`: A class that represents an animator with the provided configuration.
+     */
```

Comment:
nit: Could you use the auto-generated documentation format here? Whatever is outputted by holding down `Cmd+Shift+\`

Reply:
Same for all the multi-line docs below.

Reply:
I did that before I pushed